MAUI/Badge-View/Badge-customization.md

```diff
@@ -66,7 +66,7 @@ Content = sfBadgeView;
 
 ## Stroke customization
 
-The stroke and stroke thickness of the badge view can be customized using the [Stroke](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.BadgeSettings.html#Syncfusion_Maui_Core_BadgeSettings_StrokeProperty) and [StrokeThickness](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.BadgeSettings.html#Syncfusion_Maui_Core_BadgeSettings_BorderWidthProperty) properties, respectively.
+The stroke and stroke thickness of the badge view can be customized using the [Stroke](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.BadgeSettings.html#Syncfusion_Maui_Core_BadgeSettings_StrokeProperty) and [StrokeThickness](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.BadgeSettings.html#Syncfusion_Maui_Core_BadgeSettings_StrokeThicknessProperty) properties, respectively.
```

Comment:
@KarthikaRavichandranSF4617, the given API link is not correct for Stroke and StrokeThickness properties. Please add the proper API link.

Reply:
@naveenkumar-sanjeevirayan  bro, I have updated the proper API link for Stroke and Stroke Thickness properties.